### PR TITLE
feature/#710 added relnote for submission entity

### PR DESCRIPTION
**Pull request by toan-quach**
https://app.zenhub.com/workspaces/taipy-613ef328affde6000ff77fb3/issues/gh/avaiga/taipy-doc/710

### CODE REVIEW

File: docs/relnotes.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ This is the list of changes to Taipy releases as they were published.
     `SQLTableDataNode`, and `MongoCollectionDataNode`.<br/>
     For more information, refer to [Append new data to a data node](./manuals/core/entities/data-node-mgt.md#append-new-data-to-a-data-node).
 
+- When submitting a `Scenario^`, a `Sequence^` or a `Task^`, we will create a `Submission^` entity internally, which will hold all of the
+    metadata related to the submission such as its overall status, its submission date, the ids of the `Job^` involved, etc. The latest
+    `Submission^` of a `Scenario^`, a `Sequence^` or a `Task^` can be retrieved by calling the new `taipy.get_latest_submission()^` method.
```
Comment (jrobinAV) [inline docs/relnotes.md]:
```suggestion
- When submitting a `Scenario^`, a `Sequence^` or a `Task^`, Taipy automatically creates a
`Submission^` entity. It holds the metadata related to the submission such as its overall
submission status, its submission date, the ids of the `Job^` involved, etc. 
The latest `Submission^` of a `Scenario^`, a `Sequence^`, or a `Task^` can be retrieved by calling the new `taipy.get_latest_submission()^` method.
```

**Review by jrobinAV — APPROVED** [review]

File: docs/relnotes.md [hunk 2]
```diff
@@ -35,6 +35,11 @@ This is the list of changes to Taipy releases as they were published.
     `SQLTableDataNode`, and `MongoCollectionDataNode`.<br/>
     For more information, refer to [Append new data to a data node](./manuals/core/entities/data-node-mgt.md#append-new-data-to-a-data-node).
 
+- When submitting a `Scenario^`, a `Sequence^` or a `Task^`, Taipy automatically creates a
```
Comment (FabienLelaquais) [inline docs/relnotes.md]:
```suggestion
- A new class called `Submission^` was added. It holds the meta-data (such as its status or submission date) of all entities that are submitted: `Scenario^`, `Sequence^`, and `Task^`.</br>
The function `taipy.get_latest_submission()^` was also added to retrieve the last submitted entity.
```

Comment (FabienLelaquais) [reply docs/relnotes.md]: Globally, my point is that the paragraph should not repeat what can be found in the Reference Manual.

**Review by FabienLelaquais — APPROVED** [review]
With my proposal to replace the paragraph...

**Pull request merged**